### ISSUE TITLE
The package is not installable

### ISSUE THREAD

**Issue by gaitat** [mondeja/svg-path-bbox#100]
When I run `npm install svg-path-bbox` I get an npm error:

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path my-path\node_modules\svg-path-bbox\src\cli.cjs
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, chmod 'my-path\node_modules\svg-path-bbox\src\cli.cjs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

I am using node v14.7

**Comment by mondeja** [comment]
I have replicated it but after upgrading npm with `npm install -g npm` the binary has been fixed, hope this helps.

**Comment by scriptin** [comment]
@mondeja Still reproducible on Node v14 (but working fine on v16). The reason is a wrong file name in `package.json` "bin" section:

```
  "bin": {
    "svg-path-bbox": "src/cli.cjs"
  },
```

The file is called `src/cli.js`, not `src/cli.cjs` (not the extension). For some reason Node v16 ignores the extension.

Simply renaming the file should fix the issue. For testing, you can use [NVM](https://github.com/nvm-sh/nvm): `nvm use 14`

@gaitat I was able to find this workaround: in my package.json, add the following into "scripts":

```
"preinstall": "touch node_modules/svg-path-bbox/src/cli.cjs",
```

**Comment by gaitat** [comment]
@scriptin Thank you

**Comment by scriptin** [comment]
@gaitat Actually, my workaround doesn't work in some use cases. I've tried it in my CI/CD pipeline, and it still fails. Upgrading to Node v16 (or maybe higher, I haven't tried) is a better option.

**Comment by mondeja** [comment]
Thanks @scriptin, I'll

**Comment by mondeja** [comment]
Fixed on [v1.2.3](https://github.com/mondeja/svg-path-bbox/releases/tag/v1.2.3)

**Comment by scriptin** [comment]
@mondeja Thank you very much for such a quick response. Awesome library!
